src/modules/admin/admin.service.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  Injectable,
+  NotFoundException,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+import { JwtService } from '@nestjs/jwt';
+import { Admin } from './admin.entity';
+import { adminSigninDto } from '../../DTOs/admin-DTO/admin.dto';
+import { EncryptionService } from '../../helpers/encryption/encryption.service';
+
+@Injectable()
+export class AdminService {
+  constructor(
+    @InjectRepository(Admin) private adminRepo: Repository<Admin>,
+    private jwtService: JwtService,
+    private encryptionService: EncryptionService,
+  ) {}
+
+  async seedAdmin() {
+    try {
+      const {
+        ADMIN_EMAIL,
+        ADMIN_PASSWORD,
+        ADMIN_FIRSTNAME,
+        ADMIN_LASTNAME,
+        ADMIN_MOBILE,
+      } = process.env;
+      if (
+        !ADMIN_EMAIL ||
+        !ADMIN_PASSWORD ||
+        !ADMIN_FIRSTNAME ||
+        !ADMIN_LASTNAME ||
+        !ADMIN_MOBILE
+      ) {
+        throw new NotFoundException('Missing environmental variables');
```

Comment:
Use "Missing seed admin environment credentials" instead....

added the logic name will make debugging easy to identify which method is failing as a result of missing env... credentials